docs/configuration.md

```diff
@@ -0,0 +1,198 @@
+# Lighthouse Configuration
+
+The Lighthouse config object is the primary method of customizing a Lighthouse run to suit your use case. Using a custom config, you can limit the audits to run, add additional loads of the page under special conditions, add your own custom checks, tweak the scoring, and more.
```

Comment:
Lots of terminology in here. Throw in a link to https://github.com/GoogleChrome/lighthouse/blob/master/docs/architecture.md.